py/selenium/webdriver/common/utils.py

```diff
@@ -138,7 +137,7 @@ def keys_to_typing(value: Iterable[AnyKey]) -> List[str]:
     """Processes the values that will be typed in the element."""
     typing: List[str] = []
     for val in value:
-        if isinstance(val, Keys):
```

Comment:
This needs to be reverted. The characters need to be sent one at a time, not as a string, it's causing the 3 interactions errors that include `send_keys`

Reply:
i'm confused about how this works, though; `Keys` is just a class with a bunch of attributes, not an enum or anything, so I'm not sure how this isinstance could ever be truthy

```python
>>> from selenium.webdriver.common.keys import Keys
>>> isinstance(Keys.LEFT, Keys)
False
```

Reply:
i think [removing this clause entirely](https://github.com/SeleniumHQ/selenium/pull/10202/commits/10991584ae92f31c6f6de15186a3213181db39ab) is the right move here

Reply:
Yeah, I tried adding a few tests and seeing what it does, and I agree, short of passing in `Keys()` nothing will make it true. :)

Reply:
i think this is resolvable, too, if you agree with the change i made